packages/gnosis-relay-api/support/gnosis/docker-compose.yml

```diff
@@ -72,28 +89,31 @@ services:
     networks:
       - augur
 
-  0x-mesh:
+  0x-mesh-standalone:
     image: 0xorg/mesh:pre-release-9.0.0-wss-support
     restart: on-failure
     hostname: 0x
+    depends_on:
+      - "0x-mesh-bootstrap"
     ports:
       - 60557:60557 # rpc_port_number
-      - 60558:60558 # P2PTCPPort
-      - 60559:60559 # P2PWebSocketsPort
+      - 59558:59558 # P2PTCPPort
+      - 59559:59559 # P2PWebSocketsPort
     environment:
       - ETHEREUM_CHAIN_ID=${ETHEREUM_CHAIN_ID}
       - CUSTOM_CONTRACT_ADDRESSES=${CUSTOM_CONTRACT_ADDRESSES}
       - ETHEREUM_RPC_URL=http://geth:8545
-      - USE_BOOTSTRAP_LIST=false
+      - USE_BOOTSTRAP_LIST=true
+      - BOOTSTRAP_LIST=/dns4/0x-mesh-bootstrap/tcp/60558/ipfs/16Uiu2HAmRMgvPQV2UYKXuuCnNaFLpc36PhLp2UKVcL1ePseVcz4y,/dns4/0x-mesh-bootstrap/tcp/60559/ws/ipfs/16Uiu2HAmRMgvPQV2UYKXuuCnNaFLpc36PhLp2UKVcL1ePseVcz4y
       - BLOCK_POLLING_INTERVAL=1s
       - ETHEREUM_RPC_MAX_REQUESTS_PER_24_HR_UTC=169120 # needed when polling interval is 1s
       - VERBOSITY=4 # 5=debug 6=trace
       - WS_RPC_ADDR=0.0.0.0:60557
+      - P2P_TCP_PORT=59558
```

Comment:
Setting the ports IN the container as well so it automatically knows the right external port to use.